src/main/resources/templates/recipe/detail.html

```diff
@@ -1,22 +1,17 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="https://thymeleaf.org/">
 <head th:replace="fragments :: head"></head>
-<body class="text-center">
+<body class="text-left">
 <nav th:replace="fragments :: navigation"></nav>
 <div class="container-fluid">
 	<table class="table table-striped">
 		<tr>
 			<th>Recipe Name:</th>
 			<td th:text="${recipe.name}">Default Text</td>
-		</tr>
-		<tr>
-			<th>Recipe Categories:</th>
-			<td>
-				<th:block th:if="${recipe.categories.size() > 0}" th:each="category : ${recipe.categories}">
-					<p th:text="${category.name}"></p>
-				</th:block>
-				<p th:unless="${recipe.categories.size()}">No Categories</p>
-			</td>
+			<!--top bar if a favorite recipe-->
+			<p th:if="${recipe.favorite}">
+			<h1><span class="badge bg-warning">Favorite</span></h1>
+			</p>
```

Comment:
Switching this to the top of the table and changing the <p> tag to a <tr> tag allowed for the conditional logic to work as intended. The way it is now, the favorite badge shows on every recipe. When I switched it around it only showed on the recipe I added as favorite

Reply:
Good catch, I must have moved it while trying move something else around.